__tests__/endpoints.test.js

```diff
@@ -73,3 +73,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("GET: 200 Should respond with 200 status code and all current articles with their comment counts.", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length).toBeGreaterThan(0);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String || Number),
+          });
+          expect(article).not.toHaveProperty("body");
+        });
+      });
+  });
+  test("GET: 200 articles should be ordered by date in desending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        const sortExpected = _.orderBy(articles, ["date"], ["desc"]);
+        expect(articles).toEqual(sortExpected);
+      });
```

Comment:
I'd consider using jest-sorted to check the articles are ordered in descending order. Here are the docs for you to have a look at: https://www.npmjs.com/package/jest-sorted